egs2/TEMPLATE/asr1/db.sh

```diff
@@ -17,6 +17,8 @@ if [[ "$(hostname -d)" == clsp.jhu.edu ]]; then
                 ## Case merl :MERL setup. Neccesary directory is WAV and sdb
     LIBRISPEECH=
     VOXFORGE=
+    YESNO=
```

Comment:
`db.`sh is intend to write the path to the external corpus or the default path to put the downloaded corpus.
YESNO is very small corpus, so you don't need to write it here.